src/core/gtdatapath.c

```diff
@@ -61,6 +66,13 @@ GtStr* gt_get_gtdata_path(const char *prog, GtError *err)
       if (gt_file_exists_and_is_dir(gt_str_get(path)))
         return path;
     }
+    for (spath = GTDATA_RELATIVE_SEARCH_PATHS; *spath; spath++) {
+      had_err = gt_file_find_exec_in_path(path, prog, err);
+      if (had_err) break;
+      gt_str_append_cstr(path, *spath);
+      if (gt_file_exists_and_is_dir(gt_str_get(path)))
+        return path;
+    }
```

Comment:
I feel that `GTDATA_RELATIVE_SEARCH_PATHS ` should be searched before `GTDATA_DEFAULT_PATHS`. If there are two installations of GenomeTools, an outdated system-wide version in `/usr/local/share/genometools` and a recent version in `~/share/genometools`, I would want `~/bin/genometools` to find the relative copy before the system-wide copy.